### PR TITLE
Add image upload to rails admin

### PR DESCRIPTION
**Pull request by stellenberger**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options)
-  config.active_storage.service = :local
```
[the remark this answers is not in the document]
Comment (stellenberger) [reply config/environments/development.rb:33]: Yes, lets keep local for dev

File: config/initializers/rails_admin.rb [hunk 2]
```diff
@@ -48,6 +48,10 @@
     field :causes do
       required true
     end
+
+    # field :header_image do
+    #   partial "example_partial"
+    # end
```
[the remark this answers is not in the document]
Comment (stellenberger) [reply config/initializers/rails_admin.rb:54]: Yes, but unnecessary now as Rails Admin takes care of the image upload fields. Will remove 👍

File: app/models/campaign.rb [hunk 3]
```diff
@@ -53,6 +55,14 @@ def key_aims
     goals.where('type = ?', 'Key Aim')
   end
 
+  def header_image
```
[the remark this answers is not in the document]
Comment (stellenberger) [reply app/models/campaign.rb:58]: Yes, both will show up on the admin panel. However on endpoints, precedence will be given to header_image_s3 endpoint if an image is uploaded, shown under `header_image`

**Review by JElgar — APPROVED** [review]
Looks dreamy!

File: config/storage.yml [hunk 4]
```diff
@@ -7,28 +7,9 @@ local:
...
-# mirror:
-#   service: Mirror
-#   primary: local
-#   mirrors: [ amazon, google, microsoft ]
+amazon:
+  service: S3
+  access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
+  secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
+  region: eu-west-2
+  bucket: now-u-bucket
```
Comment (JElgar) [inline config/storage.yml:15]: Can we also pull the region and bucket name from envs

File: app/models/quiz_answer.rb [hunk 5]
```diff
@@ -1,5 +1,17 @@
 # frozen_string_literal: true
 
 class QuizAnswer < ApplicationRecord
+  include ::V2::Image::ImageService
+
+  has_one_attached :answer_image_url_s3
```
Comment (JElgar) [inline app/models/quiz_answer.rb:6]: Sorry this is really pedantic but do you think we remove "url" and "link" from these _s3 names. I feel like they aren't really links/url any more given they are s3 objects.

Comment (stellenberger) [reply app/models/quiz_answer.rb:6]: 100% agree!

**Pull request merged**